Lib/test/test_httpservers.py

```diff
@@ -29,6 +30,10 @@
 import unittest
 from test import support
 
+# Some OS accept trailingSlash as part of a filename
+# https://bugs.python.org/issue34711#msg325532 mentions Windows sometimes accepts a trailingSlash
+# so more work could be done
+trailingSlashOK = platform.system() in ('AIX')
```

Comment:
This is a string, not a one-element tuple.

Equality test may be clearer (and simple to change if needed).